core/build.gradle.kts

```diff
@@ -9,7 +9,7 @@ android {
         minSdk = AndroidConfig.minSdk
     }
 
-    namespace = "eu.kanade.tachiyomi.extension"
+    namespace = "eu.kanade.tachiyomi.lib.core"
```

Comment:
`core` is actually not a lib. Can we use `eu.kanade.tachiyomi.extension.core`? 

Also, does this affect the merging of `AndroidManifest.xml` and default icon resources?